src/app/workflow/workflow.component.ts

```diff
@@ -128,9 +127,9 @@ export class WorkflowComponent extends Entry {
       this.fireCloudURL = null;
       const version: WorkflowVersion = this.selectedVersion;
       if (version && this.isWdl(workflowRef)) {
-        this.workflowsService.wdl(workflowRef.id, version.name).subscribe(sourceFile => {
-          if (sourceFile.content && sourceFile.content.length && !this.firecloudService.wdlHasImports(sourceFile.content)) {
-            this.fireCloudURL = this.firecloudService.redirectUrl(workflowRef.full_workflow_path, version.name);
+        this.workflowsService.secondaryWdl(workflowRef.id, version.name).subscribe(sourceFiles => {
```

Comment:
Changing `sourceFiles =>` to `(sourceFiles: Array<SourceFile>) =>` might be helpful

Reply:
Changed it.